src/layout/NotificationsToastArea/NotifcationStack.tsx

```diff
@@ -24,7 +24,7 @@ type ElementProps = {
   notifications: {
     notification: Notification<any, any>;
     key: string;
-    displayData: NotificationDisplayData;
+    displayData: NotificationDisplayData | undefined;
```

Comment:
i think u want to keep this as definitely defined?

Reply:
I skimmed and it seemed like it could genuinely sometimes be undefined and it wasn't hard to make everyone handle it so it seems fine.

Reply:
are u suuuuure? i dont see it